src/features_creator/features_creator.py

```diff
@@ -77,8 +70,82 @@ def calculate_standard_deviation(data, cols):
     >>> calculate_standard_deviation(df, cols)
         week_payment1  week_payment2  week_payment3
      0              0              0              0
-
     """
     ...
+    
+def calculate_percentage_change(
+    df, pattern, compare_period=(2, 2), time_filter=None, changed_name=None
+):
+    """Calculate percentage change over a time period
```

Comment:
I think ultimately, we should move this extensive documentation out of the docstring and into the package documentation. But thanks for writing it out! And we can leave it in here for this milestone.